netbox/core/api/schema.py

```diff
@@ -277,3 +277,23 @@ def map_serializer_field(self, auto_schema, direction):
             return component.ref if component else None
         else:
             return build_basic_type(OpenApiTypes.INT)
+
+
+# Nested models can be passed by ID in requests
+# The logic for this is handled in `BaseModelSerializer.to_internal_value`
+class FixWritableNestedSerializerAllowPK(OpenApiSerializerFieldExtension):
+    target_class = 'netbox.api.serializers.BaseModelSerializer'
+    match_subclasses = True
+
+    def map_serializer_field(self, auto_schema, direction):
+        schema = auto_schema._map_serializer_field(self.target, direction, bypass_extensions=True)
```

Comment:
I think you can do a simple optimization here:
```
        if (schema := auto_schema._map_serializer_field(self.target, direction, bypass_extensions=True)) is None:
            return schema
```

Reply:
how is that better code? is that something the netbox style guide recommends?